index.html

```diff
@@ -1,110 +1,16 @@
 ---
 layout: default
 ---   
-<div class="greenhead">Where can I see the RFP?</div>
+
+{% for faq in site.data.faq %}
```

Comment:
So the one thing here is that the FAQ now becomes a global property on `site`. One thing you could think about doing is instead creating a new `layout` which consumers some page-specific FAQ. This would make the site more generalizable in terms of containing a whole bunch of different RFPs/FAQs.